src/frontend.ts

```diff
@@ -256,6 +268,31 @@ export class Frontend {
     }
   }
 
+  private translateInt(node: source.node.Int) : ReadonlyArray<IWrap<frontend.node.Node>> {
+    const { name, field, bytes, signed, littleEndian } = node;
+
+    let result = [
+      new this.implementation.node.Int(
+        new frontend.node.Int(this.id.id(name), field, bytes, signed, littleEndian, 0)
```

Comment:
Nitpick: Let's split it into multiple statements. Something like: `const tmp = new frontend.node.Int(...)`.